routing/turns_sound.hpp

```diff
@@ -56,6 +56,11 @@ class TurnsSound
   /// After the second notification has been pronounced
   /// m_nextTurnNotificationProgress == Second.
   PronouncedNotification m_nextTurnNotificationProgress;
+  /// The flag is set to true if notification about the second turn was pronounced.
```

Comment:
Which standard are you talking about?
As far as I know there's no restriction for the order in our coding standard.
We've discussed this topic recently.